### DISCUSSION TITLE
memory leak cause game to slow down and become unplayable.

### DISCUSSION THREAD

**Discussion by 40476** [FabricMC/fabric-api#3923] [Player support]
### Description

Whenever I am playing (with and without mods), the memory fills up and the game becomes unplayable and I have to restart it.
I am using openJDK 22 on linux (openSUSE tumbleweed)

**Comment by 40476** [comment]
what should I do?

**Reply by modmuss50** [reply]
Its quite hard to provide a helpful suggestion without knowing all the details. How much memory are you proving the game and what launcher are you using would be a good starting point.

**Reply by 40476** [reply]
I am using minecraft launcher and I am giving it 3GB of ram, it used to run just fine with 2GB on forge.

**Reply by 40476** [reply]
is that enough?

**Reply by 40476** [reply]
hello?
